hellosign-embedded/index.d.ts

```diff
@@ -268,3 +268,5 @@ declare module HelloSign {
 }
 
 declare var HelloSign: HelloSign.HelloSignStatic;
+
+export = HelloSign;
```

Comment:
please add to your change `export as namespace HelloSign` this will grantee that it is available both as a module and a global variable.